helvum broken on unstable

### Steps To Reproduce
Steps to reproduce the behavior:
1. build `nixpkgs#helvum`

### Build log
```
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking sources
unpacking source archive /nix/store/zvmg2k2spwir50p47r6092b0f9zgl958-source
source root is source
Executing cargoSetupPostUnpackHook
unpacking source archive /nix/store/pn1m0x87v0pvnj98a04384cnhvp3agan-helvum-0.3.4-vendor.tar.gz
Finished cargoSetupPostUnpackHook
@nix { "action": "setPhase", "phase": "patchPhase" }
patching sources
Executing cargoSetupPostPatchHook
Validating consistency between /build/source/Cargo.lock and /build/helvum-0.3.4-vendor.tar.gz/Cargo.lock
Finished cargoSetupPostPatchHook
@nix { "action": "setPhase", "phase": "configurePhase" }
configuring
meson flags: --buildtype=plain         --libdir=/nix/store/iksjk0rgqqvsy506x6zi8jx1gfzhh177-helvum-0.3.4/lib --libexecdir=/nix/store/iksjk0rgqqvsy506x6zi8jx1gfzhh177-helvum-0.3.4/libexec         --bindir=/nix/store/iksjk0rgqqvsy506x6zi8jx1gfzhh177-helvum-0.3.4/bin --sbindir=/nix/store/iksjk0rgqqvsy506x6zi8jx1gfzhh177-helvum-0.3.4/sbin         --includedir=/nix/store/iksjk0rgqqvsy506x6zi8jx1gfzhh177-helvum-0.3.4/include         --mandir=/nix/store/iksjk0rgqqvsy506x6zi8jx1gfzhh177-helvum-0.3.4/share/man --infodir=/nix/store/iksjk0rgqqvsy506x6zi8jx1gfzhh177-helvum-0.3.4/share/info         --localedir=/nix/store/iksjk0rgqqvsy506x6zi8jx1gfzhh177-helvum-0.3.4/share/locale         -Dauto_features=enabled         -Dwrap_mode=nodownload         --prefix=/nix/store/iksjk0rgqqvsy506x6zi8jx1gfzhh177-helvum-0.3.4  
The Meson build system
Version: 0.64.1
Source dir: /build/source
Build dir: /build/source/build
Build type: native build
Project name: helvum
Project version: 0.3.4
Rust compiler for the host machine: rustc -C linker=gcc (rustc 1.66.1)
Rust linker for the host machine: rustc -C linker=gcc ld.bfd 2.39
Host machine cpu family: x86_64
Host machine cpu: x86_64
Found pkg-config: /nix/store/5cip8xpjb9g10nwyzyslxpck3anc4r4h-pkg-config-wrapper-0.29.2/bin/pkg-config (0.29.2)
Run-time dependency glib-2.0 found: YES 2.74.3
Run-time dependency gtk4 found: YES 4.8.3
Run-time dependency libpipewire-0.3 found: YES 0.3.64
Program desktop-file-validate found: YES (/nix/store/w6583pwpvdnpp9kw6g8knl5lcq5219g4-desktop-file-utils-0.26/bin/desktop-file-validate)
Program appstream-util found: NO
Program cargo found: YES (/nix/store/1v5zcbfjbjvzhmn8byrwq8wd7i0nim4c-cargo-1.66.1/bin/cargo)
Message: Building in release mode
Configuring org.pipewire.Helvum.desktop using configuration
Configuring org.pipewire.Helvum.metainfo.xml using configuration
Program gtk4-update-icon-cache found: YES (/nix/store/k1dfawwm06zh4mcr82daszv5zpyk7fck-gtk4-4.8.3/bin/gtk4-update-icon-cache)
Program update-desktop-database found: YES (/nix/store/w6583pwpvdnpp9kw6g8knl5lcq5219g4-desktop-file-utils-0.26/bin/update-desktop-database)
Build targets in project: 1

helvum 0.3.4

  User defined options
    auto_features: enabled
    bindir       : /nix/store/iksjk0rgqqvsy506x6zi8jx1gfzhh177-helvum-0.3.4/bin
    buildtype    : plain
    includedir   : /nix/store/iksjk0rgqqvsy506x6zi8jx1gfzhh177-helvum-0.3.4/include
    infodir      : /nix/store/iksjk0rgqqvsy506x6zi8jx1gfzhh177-helvum-0.3.4/share/info
    libdir       : /nix/store/iksjk0rgqqvsy506x6zi8jx1gfzhh177-helvum-0.3.4/lib
    libexecdir   : /nix/store/iksjk0rgqqvsy506x6zi8jx1gfzhh177-helvum-0.3.4/libexec
    localedir    : /nix/store/iksjk0rgqqvsy506x6zi8jx1gfzhh177-helvum-0.3.4/share/locale
    mandir       : /nix/store/iksjk0rgqqvsy506x6zi8jx1gfzhh177-helvum-0.3.4/share/man
    prefix       : /nix/store/iksjk0rgqqvsy506x6zi8jx1gfzhh177-helvum-0.3.4
    sbindir      : /nix/store/iksjk0rgqqvsy506x6zi8jx1gfzhh177-helvum-0.3.4/sbin
    wrap_mode    : nodownload

Found ninja-1.11.1 at /nix/store/5p4994n7rygjg8v7jaf9dv70z3lgiymd-ninja-1.11.1/bin/ninja
meson: enabled parallel building
@nix { "action": "setPhase", "phase": "buildPhase" }
building
build flags: -j20
[0/1] Generating src/cargo-build with a custom command
   Compiling serde v1.0.136
   Compiling smallvec v1.8.0
   Compiling pkg-config v0.3.24
   Compiling libc v0.2.116


<extraneous output omitted...>


   Compiling libspa-sys v0.4.1
   Compiling pipewire-sys v0.4.1
   Compiling glib v0.15.4
   Compiling pipewire v0.4.1
error[E0425]: cannot find value `PW_KEY_NODE_TARGET` in crate `pw_sys`
   --> /build/helvum-0.3.4-vendor.tar.gz/pipewire/src/auto/keys.rs:164:28
    |
164 | key_constant!(NODE_TARGET, PW_KEY_NODE_TARGET,
    |                            ^^^^^^^^^^^^^^^^^^ help: a constant with a similar name exists: `PW_KEY_NODE_NAME`
    |
   ::: /build/source/build/src/release/build/pipewire-sys-2349f1bf3780854a/out/bindings.rs:5:8276
    |
5   | ... b"node.id\0" ; pub const PW_KEY_NODE_NAME : & [u8 ; 10usize] = b"node.name\0" ; pub const PW_KEY_NODE_NICK : & [u8 ; 10usize] = b"nod...
    |                    --------------------------------------------- similarly named constant `PW_KEY_NODE_NAME` defined here

For more information about this error, try `rustc --explain E0425`.
error: could not compile `pipewire` due to previous error
warning: build failed, waiting for other jobs to finish...
FAILED: src/helvum 
/nix/store/r1lybmy0jzjsfcmbg24h26aknbgpfvad-coreutils-9.1/bin/env CARGO_HOME=/build/source/build/cargo-home /nix/store/1v5zcbfjbjvzhmn8byrwq8wd7i0nim4c-cargo-1.66.1/bin/cargo build --manifest-path /build/source/Cargo.toml --target-dir /build/source/build/src --release && cp src/release/helvum src/helvum
ninja: build stopped: subcommand failed.
```

### Additional context
 - Upstream issue: https://gitlab.freedesktop.org/pipewire/helvum/-/issues/76
 - Last successful build: https://hydra.nixos.org/build/204657516
 - First failing build: https://hydra.nixos.org/build/205185819

### Notify maintainers
 - @fufexan (package maintainer)
 - @jtojnar @Kranzes (pipewire package maintainers)

--- a/pkgs/applications/audio/helvum/default.nix
+++ b/pkgs/applications/audio/helvum/default.nix
@@ -16,20 +16,20 @@
 
 stdenv.mkDerivation rec {
   pname = "helvum";
-  version = "0.3.4";
+  version = "0.4.0";
 
   src = fetchFromGitLab {
     domain = "gitlab.freedesktop.org";
     owner = "pipewire";
     repo = pname;
     rev = version;
-    sha256 = "0nhv6zw2zzxz2bg2zj32w1brywnm5lv6j3cvmmvwshc389z2k5x1";
+    hash = "sha256-TvjO7fGobGmAltVHeXWyMtMLANdVWVGvBYq20JD3mMI=";
   };
 
   cargoDeps = rustPlatform.fetchCargoTarball {
     inherit src;
     name = "${pname}-${version}";
-    hash = "sha256-EIHO9qVPIXgezfFOaarlTU0an762nFmX1ELbQuAZ7rY";
+    hash = "sha256-W5Imlut30cjV4A6TCjBFLbViB0CDUucNsvIUiCXqu7I=";
   };
 
   nativeBuildInputs = [
@@ -49,9 +49,6 @@ stdenv.mkDerivation rec {
     gtk4
     pipewire
   ];
-
-  # FIXME: workaround for Pipewire 0.3.64 deprecated API change, remove when fixed upstream
-  NIX_CFLAGS_COMPILE = [ "-DPW_ENABLE_DEPRECATED" ];
 
   meta = with lib; {
     description = "A GTK patchbay for pipewire";

